pkgs/tools/security/gallia/default.nix

```diff
@@ -29,22 +32,19 @@ python3.pkgs.buildPythonApplication rec {
     construct
     msgspec
     pydantic
+    pygit2
     tabulate
     tomlkit
-    xdg
     zstandard
   ];
 
+  SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
+
   checkInputs = with python3.pkgs; [
     pytestCheckHook
+    pytest-asyncio
   ];
 
-  postPatch = ''
-    substituteInPlace pyproject.toml \
```

Comment:
At least `msgspec` has to be adjusted. We already have  `msgspec 0.11.0`.

Reply:
I could update this upstream (I am the maintainer of `gallia`), then this change can be avoided.